sympy/physics/optics/utils.py

```diff
@@ -460,15 +460,28 @@ def lens_makers_formula(n_lens, n_surr, r1, r2):
         Radius of curvature of first surface.
     r2 : sympifiable
         Radius of curvature of second surface.
+    d : sympifiable
+        Thickness of lens.
+        Applicable when lens = 'thick'
+    lens : string, default 'thin'
+        Takes value 'thin' for thin lens, 'plano' for
+        plano lens and 'thick' for thick lens
 
     Examples
     ========
 
     >>> from sympy.physics.optics import lens_makers_formula
     >>> lens_makers_formula(1.33, 1, 10, -10)
     15.1515151515151
+    >>> lens_makers_formula(1.2, 1, 10,lens='plano')
+    50.0000000000000
+    >>> lens_makers_formula(1.33, 1, 10, -10, d=1, lens='thick')
+    15.3418463277618
 
     """
+    valid_lens = ['thin','thick','plano']
```

Comment:
Better a tuple rather than a list so it's a compile time constant.

Reply:
I'll change that